content/docs/deploy/enterprise.md

```diff
@@ -1,46 +1,109 @@
 ---
-title: About
-description: What does the Pomerium Enterprise Console offer?
+title: Pomerium Enterprise
+description: Learn what features come with Pomerium Enterprise, including a Console GUI where you can manage your policies, namespaces, groups, routes, and more.
+keywords:
+  [
+    Pomerium Enterprise,
+    PPL Builder,
+    Console GUI,
+    namespaces,
+    directory sync,
+    device management,
+    groups,
+    programmatic api,
+    branding,
+  ]
 ---
 
+import ClearIcon from '@mui/icons-material/Clear';
+
 # Pomerium Enterprise
 
-Pomerium Enterprise is built on Pomerium Core (Open Source). Pomerium Enterprise makes Pomerium easier to manage at scale, and adds additional functionality aimed at organizations with auditing, compliance, governance, and risk management needs.
+Pomerium Enterprise is built on our open-source Pomerium Core offering. Pomerium Enterprise makes Pomerium easier to manage at scale, and adds additional functionality aimed at organizations with auditing, compliance, governance, and risk management needs.
+
+## Pomerium Enterprise Features
+
+Pomerium Enterprise comes with all the capabilities in Pomerium Core, plus the following features:
+
+### Enterprise Console
+
+The **Enterprise Console** provides a dashboard where you can view traffic and logs, manage routes and policies, import external data, configure global and namespaced settings, and more.
```

Comment:
Let's double-check whether you can still view logs in the Console.

Reply:
I discussed this @desimone and he said we refer to the **Deployments** tab as a way for admins to provide auditing functionality since it can reveal who is doing what in the control plane.

Reply:
There's also a reference to "access logs" below. Is that accurate?

Reply:
Hmm I think it's just change logs. I can remove "access" from the explanation below.